app/gulp/js/questions.js

```diff
@@ -1,23 +1,24 @@
 // Code to control adding/removing answer choices
-onPage('questions', 'edit', () => {
-  let num = 10000; // We only support 10000 answers per question, get over it
-  $('#add-answer').click((e) => {
-    num -= 1;
-    e.preventDefault();
-    // Regexes here modify the duplicated div to create a unique hash
-    // (by decrementing 1 from 10000)
-    let newAnswer = $('#wrong-answers div:first-child').prop('outerHTML');
-    newAnswer = newAnswer.replace(/_[0-9]+_/g, `_${num}_`);
-    newAnswer = newAnswer.replace(/\[[0-9]+\]/g, `[${num}]`);
-    newAnswer = newAnswer.replace(/value="*."/g, 'value=""');
-    $('#wrong-answers').append(newAnswer);
-  });
+// onPage('questions', 'edit', () => {
```

Comment:
You could also do something like
```js
const answerAddRemove = () => {
  // ... handler code here ...
};

onPage('question', 'edit', answerAddRemove);
onPage('question', 'new', answerAddRemove);
```
but this works fine too, if there isn't an '#add-answer' on a different page that does something else.